src/main/java/org/sonarlint/intellij/analysis/SonarLintAnalysisConfigurator.java

```diff
@@ -47,12 +48,14 @@
 import java.util.Properties;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+
 import org.apache.commons.lang.StringUtils;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.sonar.runner.api.IssueListener;
-import org.sonarlint.intellij.config.SonarLintProjectSettings;
-import org.sonarlint.intellij.console.SonarLintConsole;
+import org.sonarlint.intellij.ui.SonarLintConsole;
+import org.sonarlint.intellij.test.TestRunnerFacade;
+import org.sonarlint.intellij.util.SonarLintUtils;
 
 public class SonarLintAnalysisConfigurator {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1118)